.github/jobs/win32.yml

```diff
@@ -62,6 +66,23 @@ jobs:
 
         displayName: 'Enable Crash Dumps'
 
+      - powershell: |
+          $vs = vswhere -latest -requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64 -property installationPath
+          $msvc = Get-ChildItem "$vs\VC\Tools\MSVC" | Sort-Object Name -Descending | Select-Object -First 1
+          $asan = "$($msvc.FullName)\bin\Hostx64\x64"
+
+          $destinations = @(
+              "build${{ variables.solutionName }}\Apps\Playground\RelWithDebInfo",
+              "build${{ variables.solutionName }}\Apps\UnitTests\RelWithDebInfo"
+          )
+
+          foreach ($dest in $destinations) {
+              Get-ChildItem "$asan\clang_rt.asan_*.dll" |
+                  Copy-Item -Destination $dest
+          }
+        displayName: "Copy ASAN runtime DLLs"
+        condition: ${{ parameters.enableSanitizers }}
```

Comment:
This is fine for now. Consider doing this inside CMake since it seems like it's part of the build.

Reply:
Hmm, maybe we should do the same and add them to the path? Do you know what adds them to the path?